services/kaze/controllers/application.go

```diff
@@ -228,29 +230,27 @@ func TransferApplicationOwnership(c *gin.Context) {
 
 // FetchMetrics retrieves the metrics of an application's container
 func FetchMetrics(c *gin.Context) {
+	c.Writer.Header().Set("Content-Type", "text/event-stream")
 	appName := c.Param("app")
-	filter := utils.QueryToFilter(c.Request.URL.Query())
-	var timeSpan int64
-
-	for unit, converter := range timeConversionMap {
-		if val, ok := filter[unit].(string); ok {
-			timeVal, err := strconv.ParseInt(val, 10, 64)
-			if err != nil {
-				continue
-			}
-			timeSpan += timeVal * converter
-		}
-	}
+	metricsFetchInterval := c.Query("fetch_interval")
+	metricsCount := c.Query("fetch_count")
```

Comment:
In case these query params are not set, you should have some default values in place.